watertap/tools/oli_api/util/tests/test_periodic_table.py

```diff
@@ -0,0 +1,29 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+#################################################################################
+
+import pytest
+from pathlib import Path
+from os.path import join
+from pandas import read_csv
+
+
+@pytest.fixture
+def get_test_file_columns():
+    test_file = read_csv(join(Path(__file__).parents[1], "periodic_table.csv"))
+    return test_file.columns
+
+
+@pytest.mark.unit
```

Comment:
we should add checks to verify size (i.e., number of rows, number of columns) and perhaps any other checks we'd want, just in case someone else came along and altered the file for any reason whatsoever.

Reply:
@adam-a-a I added a few tests based on your suggestions